assets/scripts/index.js

```diff
@@ -0,0 +1,126 @@
+function toggleMenuFunction () {
+  let burger = document.querySelector(".js-nav-toggle");
+  let burgerMenu = document.querySelector(".njs-nav-menu");
+  let mainNav = document.querySelector(".js-nav");
+  burger.classList.toggle("open");
+  burgerMenu.classList.toggle("open");
+  mainNav.classList.toggle("open");
+}
+
+function toTop () {
+  if (window.scrollTo) {
+    window.scrollTo( {top:0, behavior:"smooth"} )
+  }else{
```

Comment:
@edwartexe don't forget that blank spaces exist! 😄 
```suggestion
  } else {
```